tinymce6-parent/tinymce6/src/main/java/wicket/contrib/tinymce6/TinyMceInitializer.java

```diff
@@ -0,0 +1,19 @@
+package wicket.contrib.tinymce6;
```

Comment:
I guess these classes are copy/paste from v3/v4 ?
I think it is a good time to rename the package to `org.wicketstuff.tinymce6`